### PR TITLE
update helidon doc

### PR DESCRIPTION
**Pull request by bhavaniravichandran**

### CODE REVIEW

File: content/en/docs/samples/hello-helidon/_index.md [hunk 1]
```diff
@@ -11,20 +11,25 @@ Install Verrazzano by following the [installation]({{< relref "/docs/setup/insta
...
 ## Deploy the Hello World Helidon application
 
+1. To run this application in the default namespace.
+   ```
+   $kubectl label namespace default hello-helidon verrazzano-managed=true istio-injection=enabled
+   ```
+   For the rest of this document, it is assumed that the application is running in a namespace. If it runs in a default namespace, the -n option in the following commands can be ignored.
 
-1. Create a namespace for the application and add a label identifying the namespace as managed by Verrazzano.
+2. To run the application in a namespace other than default namespace, create a namespace for the application and add a label identifying the namespace as managed by Verrazzano.
```
Comment (rosemarymarano) [inline content/en/docs/samples/hello-helidon/_index.md:20]: other than default namespace -> other than the default namespace

File: content/en/docs/samples/hello-helidon/_index.md [hunk 2]
```diff
@@ -11,20 +11,25 @@ Install Verrazzano by following the [installation]({{< relref "/docs/setup/insta
 
 ## Deploy the Hello World Helidon application
 
+1. To run this application in the default namespace.
+   ```
+   $kubectl label namespace default hello-helidon verrazzano-managed=true istio-injection=enabled
+   ```
+   For the rest of this document, it is assumed that the application is running in a namespace. If it runs in a default namespace, the -n option in the following commands can be ignored.
```
Comment (rosemarymarano) [inline content/en/docs/samples/hello-helidon/_index.md:18]: If it runs in a default namespace -> If it runs in the default namespace
the -n option in the following commands can be ignored -> you can ignore the -n option in the following commands.

**Review by rosemarymarano — COMMENTED** [review]
These edits, plus you don't have to change the numbering from 1. to other numbers. They appear correctly in the Hugo docs, https://verrazzano.io/latest/docs/samples/hello-helidon/

File: content/en/docs/samples/hello-helidon/_index.md (hunk 2 above)
Comment (markxnelson) [reply content/en/docs/samples/hello-helidon/_index.md:18]: that "hello-helidon" after "default" should not be there

Comment (markxnelson) [reply content/en/docs/samples/hello-helidon/_index.md:18]: put the -n in backticks

File: content/en/docs/samples/hello-helidon/_index.md (hunk 1 above)
Comment (bhavaniravichandran) [reply content/en/docs/samples/hello-helidon/_index.md]: fixed

File: content/en/docs/samples/hello-helidon/_index.md (hunk 2 above)
Comment (bhavaniravichandran) [reply content/en/docs/samples/hello-helidon/_index.md]: fixed

**Review by rosemarymarano — APPROVED** [review]

File: content/en/docs/samples/bobs-books.md [hunk 3]
```diff
@@ -40,6 +40,12 @@ For more information and the source code of this application, see the [Verrazzan
 
 ## Deploy the example application
 
+1. To run this application in the default namespace.
+   ```
+   $kubectl label namespace default verrazzano-managed=true istio-injection=enabled
+   ```
+   For the rest of this document, it is assumed that the application is running in a namespace. If it runs in the default namespace, you can ignore the ```-n``` option in the following commands.
```
Comment (rosemarymarano) [inline content/en/docs/samples/bobs-books.md:47]: running in a namespace. -> running in a namespace other than `default`.

@bhavaniravichandran @markxnelson Sorry to add an edit after I approved this PR, but this wording kept troubling me for two reasons: first, it isn't entirely clear when you say, "it is assumed that the application is running in a namespace" that you mean a named namespace, right? In the preceding example, `$kubectl label namespace default verrazzano-managed=true istio-injection=enabled` the namespace is `default`, right? In all the following steps, they give a named namespace, `$ kubectl create namespace bobs-books`. Second, we tell them in the first step how to run this application in the default namespace and then, in every step thereafter, provide the namespace `bobs-books`, which they can ignore if they are running in the default namespace. I think that between step 1 and step 2 the instructions are confusing. If you agree, I'd be glad to provide input for a way to explain both choices without the confusion.

Comment (bhavaniravichandran) [reply content/en/docs/samples/bobs-books.md:47]: Would love for you to fix it. Thanks.

**Review by rosemarymarano — APPROVED** [review]
Much better!!! Thanks.

**Review by markxnelson — APPROVED** [review]

**Pull request merged**